tripy/nvtripy/frontend/tensor.py

```diff
@@ -223,11 +235,26 @@ def eval(self) -> runtime.MemRefValue:
                 f"Note: Tensor was evaluated while compiling here: {str_from_stack_info(self.trace_tensor.eval_stack_info)}",
                 mode="once",
             )
+        return self
 
-        return data
+    def tolist(self) -> Union[List, numbers.Number]:
+        """
+        Returns the tensor as a nested list. If the tensor is a scalar, returns a python number.
+
+        Returns:
+            The tensor represented as a nested list or a python number.
+
+        .. code-block:: python
+        :linenos:
+        :caption: eval
+
+            tensor = tp.ones((2, 2))
+            tensor_list = tensor.tolist()
```

Comment:
We'll need to add a `# doc: print-locals tensor_list` (and also include `tensor` if you want that as well).